.github/workflows/insider-release.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish insider release on each push to main branch
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'npm'
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: npm ci
+      - run: npm run build
+
+      - name: Resolve version
+        id: vars
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+      - name: 'Version based on commit: 0.0.0-insider.${{ steps.vars.outputs.sha_short }}'
+        run: npm version -w libraries 0.0.0-insider.${{ steps.vars.outputs.sha_short }} --force --no-git-tag-version
+
+      - name: 'rebuild ui-library-angular to reflect version in libraries/ui-library-angular/dist/ui-library-angular/package.json'
+        run: npm run build -w libraries/ui-library-angular
+
+      - name: publish
+        run: |
+          npm publish --tag=insider --access=public libraries/ui-library/
```

Comment:
Would it be an option to add the hash to the insider tag? 
Otherwise we would have at some point a ton of "insider" tags